src/cascadia/TerminalSettingsModel/FileUtils.cpp

```diff
@@ -179,18 +179,23 @@ namespace winrt::Microsoft::Terminal::Settings::Model
                 buffer.erase(0, Utf8Bom.size());
             }
 
+            if (lastWriteTime)
+            {
+                THROW_IF_WIN32_BOOL_FALSE(GetFileTime(file.get(), nullptr, nullptr, lastWriteTime));

```

Comment:
`THROW`? or `LOG`? Can this reasonably fail?

Reply:
don't think it's much worth worrying about in this case.. hopefully

Reply:
Oh right I forgot to respond that I think throwing is the right thing to do here. A caller should be able to trust us that we either successfully fill all out parameters or return a failure (exception).

Reply:
But does this leave the opportunity for an unguarded exception that takes down the app? "Caller" is one thing, but "Terminal the app and its reliability" is another

Reply:
`ReadFile` in the same function throws exceptions as well.